components/eam/src/dynamics/se/stepon.F90

```diff
@@ -232,13 +236,14 @@ subroutine stepon_run2(phys_state, phys_tend, dyn_in, dyn_out )
    use cam_history,     only: outfld, hist_fld_active
    use prim_driver_base,only: applyCAMforcing_tracers
    use prim_advance_mod,only: applyCAMforcing_dynamics
-   use element_ops,     only: get_temperature
+   use element_ops,     only: get_temperature, get_pot_vort
 
    type(physics_state), intent(inout) :: phys_state(begchunk:endchunk)
    type(physics_tend),  intent(inout) :: phys_tend(begchunk:endchunk)
    type (dyn_import_t), intent(inout) :: dyn_in  ! Dynamics import container
    type (dyn_export_t), intent(inout) :: dyn_out ! Dynamics export container
    real(r8) :: temperature(np,np,nlev)   ! Temperature from dynamics
+   real(r8) :: potvort(np,np,nlev)      ! Potential vorticity from dynamics
```

Comment:
Given that the lines below are now commented, should we just remove all the mods to this file?

Reply:
Yes, I'll do that now.